src/app/firedancer-dev/main.c

```diff
@@ -104,9 +104,9 @@ extern fd_topo_run_tile_t fd_tile_archiver_writer;
 extern fd_topo_run_tile_t fd_tile_archiver_playback;
 extern fd_topo_run_tile_t fd_tile_shredcap;
 
-extern fd_topo_run_tile_t fd_tile_bencho;
-extern fd_topo_run_tile_t fd_tile_benchg;
-extern fd_topo_run_tile_t fd_tile_benchs;
+extern fd_topo_run_tile_t fd_tile_snapshot_restore_SnapRd;
```

Comment:
```suggestion
extern fd_topo_run_tile_t fd_tile_snaprd;
```
Consistency more important than special naming here

Reply:
[44e5b38](https://github.com/firedancer-io/firedancer/pull/5323/commits/44e5b3855ea558a104d7cc6332cceac90fb0f1ed)